app/experimenter/experiments/constants.py

```diff
@@ -65,6 +65,10 @@ def FEATURE_TYPE_CHOICES(cls):  # pragma: no cover
 
     RAPID_TYPE_CHOICES = ((RAPID_AA_CFR, "Rapid CFR A/A Experiment"),)
 
+    RAPID_FEATURE_CHOICES = (("FEATURE 1", "FEATURE 1"), ("FEATURE 2", "FEATURE 2"))
+
+    RAPID_AUDIENCE_CHOICES = (("AUDIENCE 1", "AUDIENCE1"), ("AUDIENCE 2", "AUDIENCE 2"))
```

Comment:
Should this be `AUDIENCE1` or `AUDIENCE 1` (with a space)?
Or is it irrelevant because they're placeholders?

Reply:
they're placeholder values, which will be replaced